src/codec/framed_write.rs

```diff
@@ -247,7 +247,10 @@ where
                             // If *only* the CONTINUATION frame header was
                             // written, and *no* header fields, we're stuck
                             // in a loop...
-                            panic!("CONTINUATION frame write loop; header value too big to encode");
```

Comment:
Wanna leave a `debug!` (should it be `warn!`?) with a similar message, so users can at least see *why* this header was too big? Maybe it's not important...

Reply:
Yeah, added `warn` given this issue is technically is still an unresolved bug.